Add default errorHandler() to vault-backend (#14517)
backstage/backstage: .changeset/rude-mayflies-heal.md

---
'@backstage/plugin-vault-backend': patch
---

> Added errorHandler() middleware to vault-backend to prevent errors to cause a crash

Reviewer: just nitpicking - optional suggestion

```suggestion
Added `errorHandler()` middleware to `router` to prevent crashes caused by fatal errors in plugin backend
```

`vault-backend` context is clear here as it would be listed in the changeset section for the `@backstage/plugin-vault-backend`
